common/fileview2/fvwugen.cpp

```diff
@@ -54,12 +54,12 @@ MODULE_INIT(INIT_PRIORITY_STANDARD)
 IHqlExpression * addFilter(IHqlExpression * dataset, IHqlExpression * limitField)
 {
     IHqlExpression * lower = createConstant(limitField->queryType()->castFrom(true, (__int64)0));
-    lower = createValue(no_colon, lower, createValue(no_stored, createConstant(LOWER_LIMIT_ID)));
+    lower = createValue(no_colon, lower->getType(), lower, createValue(no_stored, makeVoidType(), createConstant(LOWER_LIMIT_ID)));
```

Comment:
Why does createConstant(LOWER_LIMIT_ID) have void type?

Reply:
It doesn't. It is the stored workflow action that has the void type, with a parameter of LOWER_LIMIT_ID.   E.g. see equivalent entries in the grammar.